src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -760,7 +760,7 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
   </data>
   <data name="Info_AddPkgCPM" xml:space="preserve">
     <value>PackageReference for package '{0}' added to '{1}' and PackageVersion added to central package management file '{2}'.</value>
-    <comment>0 - The package ID. 1 - Directory.Packages.props file path. 2 - Project file path.</comment>
+    <comment>0 - The package ID. 1 - Project file path. 2 - Directory.Packages.props.</comment>
```

Comment:
I think the reason there was no updates to the `.Designer.cs` file is because I only updated the comments for the resource.  The swap for the argument actually happens from the caller side.